### DISCUSSION TITLE
blog/come-non-gestire-la-fama

### DISCUSSION THREAD

**Discussion by giscus** [peppocola/pompelmo#6] [Q&A]
# blog/come-non-gestire-la-fama

Il potere incredibile del post su Facebook del Comune di Gravina mi ha portato ad essere intasato di post, messaggi, telefonate, interviste, articoli... sto impazzendo.

https://www.pompelmo.xyz/blog/come-non-gestire-la-fama

**Comment by ProtaDec** [comment]
Io che correrò a vedere i commenti degli haters immediatamente

**Reply by peppocola** [reply]
C'è gente che si lamenta dei cervelli in fuga (ma cosa volete? la nasa a gravina??) 
Altri che dicono che ora gravina sale sul carro del vincitore ma non ha meriti (ma io tutte le scuole le ho fatte a gravina, cosa dovrebbero fare? l'università di gravina??)
In particolare uno di questi è stato cancellato dopo la mia risposta a tono 🗡️ 
E il mitico bro "e sticazzi????" Quello mi ha fatto molto ridere
